doc/radosgw/multisite.rst

```diff
@@ -617,7 +617,7 @@ Failover and Disaster Recovery
 If the master zone should fail, failover to the secondary zone for
 disaster recovery.
 
-1. Make the secondary zone the master and default zone. For example:
+#. Make the secondary zone the master and default zone. For example:
```

Comment:
i am not sure i am following you. so it's more about consistency. and how come this is related to `git log`?